CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2021-04-22
+
+This release is rolling out to all regions - ETA for conclusion 2021-04-26 for public cloud and 2021-05-03 for sovereign clouds.
+
+### Announcements
+
+* Kubernetes version 1.17 has now been deprecated since March 31st.
+* CSI Drivers will become default for [Kubernetes versions 1.21+](https://docs.microsoft.com/azure/aks/csi-storage-drivers).
+* Previous [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) deprecation was June 30th 2021. To better align with Kubernetes Upstream pod security policy (preview) deprecation will begin with Kubernetes version 1.21, with its removal in version 1.25. As Kubernetes Upstream approaches that milestone, the Kubernetes community will be working to document viable alternatives.
+* For all AKS clusters using Kubernetes v1.20+, CoreDNS will be upgraded to version 1.8.3. This will remove `resyncperiod` and `upstream`from the Kubernetes plugin.
+
+### Release Notes
+
+* New Features
+  * You can now update Windows passwords via Azure cli.
+* Bug Fixes
+  * Fixed a bug with Cert Rotation trying to call windows agentpools, this is a linux only function.
```

Comment:
> this is a linux only function

this is confusing. does it mean cert rotation doesn't apply to Windows? is it documented somewhere?

Reply:
Yes cert rotation does not apply to windows.  I will updating the doc to reflect that.